### ISSUE TITLE
BaseViewHolder Deprecated, what is the alternative?

### ISSUE THREAD

**Issue by amankumarjain** [DevAhamed/MultiViewAdapter#34]

**Comment by DevAhamed** [comment]
com.ahamed.multiviewadapter.ItemViewHolder

I forget to update the doc. Thanks for pointing out.

**Comment by amankumarjain** [comment]
please update it asap, i'm stuck with it.

**Comment by zunjae** [comment]
Still not updated

**Comment by Kelvao** [comment]
Still not updated²

**Comment by DevAhamed** [comment]
I was not able to focus on the library for the past few months. I will be working on it in the next few weeks

**Comment by mangeshsambare** [comment]
Hi 
Any updates??

**Comment by zunjae** [comment]
It's best to just pull it down or put a notice.

**Comment by mangeshsambare** [comment]
@zunjae Did you pull & solved this issue? If you have solved then let me know the procedure. I am newbie here. So don't know about too much on github. I will be grateful for your help.

**Comment by zunjae** [comment]
No, I dropped this library as there are better and more active ones out there. Hint: use google.
